openassessment/xblock/test/test_staff_area.py

```diff
@@ -781,3 +783,46 @@ def _create_submission(item, values, types):
         peer_api.on_start(submission["uuid"])
         workflow_api.create_workflow(submission["uuid"], types)
         return submission
+
+    @ddt.data('files_names', 'files_name')
+    @scenario('data/self_only_scenario.xml', user_id='Bob')
+    def test_file_name_is_rendered_on_template(self, xblock, key):
+        """
+        This test checks if file name is rendered on Manage-individual-learner template ie oa_student_info.html.
```

Comment:
something like `Validates the file name is visible to staff when viewing individual submission`. Again, no need to mention the HTML file or the button name.